provisioning/Samples/service/EnrollmentGroupSample/EnrollmentGroupSample.csproj

```diff
@@ -2,12 +2,13 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <LangVersion>8.0</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Devices.Provisioning.Service" Version="1.17.2" />
+    <PackageReference Include="CommandLineParser" Version="2.8.0" />
   </ItemGroup>

```

Comment:
This may need a temporary reference to DotNetty.Common. See https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/main/provisioning/Samples/device/SymmetricKeySample/SymmetricKeySample.csproj#L19.

You'll know if the console app won't exit.

Reply:
> This may need a temporary reference to DotNetty.Common. See https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/main/provisioning/Samples/device/SymmetricKeySample/SymmetricKeySample.csproj#L19.
> 
> You'll know if the console app won't exit.

Added